[Diversity] Add Voting Logic Examples

zanbaldwin opened the pull request:
Add reStructed Text table listing all possible combinations that would result in an actionable item passing the voting stage.

zanbaldwin commented on contributing/diversity/governance.rst:
```diff
@@ -72,6 +72,23 @@ For an actionable item to pass, approval from greater than 50% of the voting key
+combinations of votes will result in an actionable item passing:
+
++-----+---------+---------+------------------------------+
+| For | Against | Abstain | Relative Approval Percentage |
```
Anyone have any opinions on whether the final column, _Relative Approval Percentage_, is necessary or not?

TimoBakx replied:
I think we can skip it.

zanbaldwin committed:
[Diversity] Add Voting Logic Examples

Add reStructed Text table listing all possible combinations that would result in an actionable item passing the voting stage.

TimoBakx reviewed (approved):
The math works out. Looks great and clear. Nice work!

javiereguiluz commented:
Thanks Zan.

HeahDude commented on contributing/diversity/governance.rst:
```diff
@@ -72,6 +72,23 @@ For an actionable item to pass, approval from greater than 50% of the voting key
+| 4   | 0       | 1       |
++-----+---------+---------+
+| 3   | 1       | 1       |
++-----+---------+---------+
```
What about 3-0-2?

zanbaldwin replied:
3-0-2 would mean that the quorum would be 60% (less than the required 75%).
